src/pages/billing/List.js

```diff
@@ -307,7 +292,7 @@ export const BillingList = ajaxCaller(class BillingList extends Component {
     const { billingProjects } = this.state
 
     if (_.some({ creationStatus: 'Creating' }, billingProjects) && !this.interval) {
-      this.interval = setInterval(() => this.refresh(), 10000)
+      this.interval = setInterval(() => this.loadProjects(), 10000)
```

Comment:
@zarsky-broad for some reason this polling isn't getting set after I create a project. Do you know why that is?